EntityFramework.Exceptions.Tests/Tests.csproj

```diff
@@ -48,6 +48,10 @@
     <PackageReference Include="Oracle.EntityFrameworkCore" Version="8.21.121" />
     <PackageReference Include="Pomelo.EntityFrameworkCore.MySql" Version="8.0.2" />
   </ItemGroup>
+  
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net8.0'">
+    <DefineConstants>BULK_OPERATIONS</DefineConstants>
```

Comment:
Is this required because `ExecuteUpdate` and `ExecuteDelete` were added in EF Core 8?

Reply:
Correct. I thought about just checking the .NET version in the test file itself, but I thought a named constant would make it more clear why certain tests are only run for .NET 8.

Reply:
I dropped support for .Net 6 and EFCore 6 in the latest commit, can you remove that constant?